specification/containerservice/resource-manager/readme.go.md

```diff
@@ -52,6 +53,16 @@ batch:
   - tag: package-2017-07
 ```
 
+### Tag: package-2022-01-02-preview and go
+
+These settings apply only when `--package-2022-01-02-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2022-01-02-preview' && $(go)
+namespace: containerservice
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2022-01-02-preview/$(namespace)
```

Comment:
```suggestion
output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2022-01-02-preview/$(namespace)
```

Reply:
@ArcturusZhang  Thanks for pointing this out. Have fixed in commit [3cace709](https://github.com/Azure/azure-rest-api-specs/pull/17372/commits/3cace709f9d5be9e6038aeaec4290c7b1d54f162). But the track 1 go sdk pipeline is still failing.

Reply:
After updating the tag definition in `readmd.go.md` to be the same as the default tag in `readmd.md`, the issue is solved.